qemu/tests/cfg/block_iscsi_with_specical_max_sectors.cfg

```diff
@@ -0,0 +1,50 @@
+- block_iscsi_with_specical_max_sectors:
+    virt_test_type = qemu
+    type = block_iscsi_with_specical_max_sectors
+    required_qemu = [6.2.0,)
+    only Linux
+    start_vm = no
+    kill_vm = yes
+    kill_vm_gracefully = yes
+    shutdown_cleanly = no
+    data_images="stg1 stg2 stg3 stg4"
+    images += " ${data_images}"
+    drive_werror = stop
+    drive_rerror = stop
+    drive_format_stg1 = scsi-hd
+    image_format_stg1 = raw
+    image_raw_device_stg1 = yes
+    drive_format_stg2 = scsi-hd
+    image_format_stg2 = qcow2
+    image_raw_device_stg2 = yes
+    drive_format_stg3 = virtio
+    image_format_stg3 = raw
+    image_raw_device_stg3 = yes
+    drive_format_stg4 = virtio
+    image_format_stg4 = qcow2
+    image_raw_device_stg4 = yes
+    vg_name = vg-sector
+    lv_stg1 = disk1
+    lv_stg2 = disk2
+    lv_stg3 = disk3
+    lv_stg4 = disk4
+    lvs_name = "${lv_stg1},${lv_stg2},${lv_stg3},${lv_stg4}"
+    image_name_stg1 = /dev/${vg_name}/${lv_stg1}
+    image_name_stg2 = /dev/${vg_name}/${lv_stg2}
+    image_name_stg3 = /dev/${vg_name}/${lv_stg3}
+    image_name_stg4 = /dev/${vg_name}/${lv_stg4}
+    blk_extra_params_stg4 = "serial=${lv_stg4}"
+    blk_extra_params_stg1 = "serial=${lv_stg1}"
+    blk_extra_params_stg2 = "serial=${lv_stg2}"
+    blk_extra_params_stg3 = "serial=${lv_stg3}"
+    target = iqn.2019-12.com.redhat:kvm-qe
```

Comment:
This may be not permitted for the target info looks like some 'private' information, we can use --target/--portal_ip options instead to pass the iscsi confs.

Reply:
This just set the Initiatorname and target for the local private iscsi server. 
I do not get your idea, could you please share how to config it in cfg  ?

Reply:
python ConfigTest.py --target